test/e2e/network/conntrack.go

```diff
@@ -268,13 +269,136 @@ var _ = SIGDescribe("Conntrack", func() {
 		// Check that the second pod keeps receiving traffic
 		// UDP conntrack entries timeout is 30 sec by default
 		ginkgo.By("checking client pod connected to the backend 2 on Node IP " + serverNodeInfo.nodeIP)
-		if err := wait.PollImmediate(5*time.Second, time.Minute, logContainsFn(podBackend2)); err != nil {
+		if err := wait.PollImmediate(5*time.Second, time.Minute, logContainsFn(podBackend2, podClient)); err != nil {
 			logs, err = e2epod.GetPodLogs(cs, ns, podClient, podClient)
 			framework.ExpectNoError(err)
 			framework.Logf("Pod client logs: %s", logs)
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	// Regression test for #74839, where:
+	// Packets considered INVALID by conntrack are now dropped. In particular, this fixes
+	// a problem where spurious retransmits in a long-running TCP connection to a service
+	// IP could result in the connection being closed with the error "Connection reset by
+	// peer"
+	// xref: https://kubernetes.io/blog/2019/03/29/kube-proxy-subtleties-debugging-an-intermittent-connection-reset/
+	ginkgo.It("should drop INVALID conntrack entries", func() {
+		serverLabel := map[string]string{
+			"app": "boom-server",
+		}
+
+		serverPod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "boom-server",
+				Labels: serverLabel,
+			},
+			Spec: v1.PodSpec{
+				NodeName: serverNodeInfo.name,
+				Containers: []v1.Container{
+					{
+						Name:  "boom-server",
+						Image: "aojea/conntrack:invalid",
```

Comment:
the image will be updated?

Reply:
yeah, you can see that this is added in a different commit 93c513a1436e5adbad3f6cd140adf8b2c30e6649, but I wanted to be sure that the new tests works in the CI

Reply:
Can you rebase it into this PR so that they will be merged together?